flow/designs/ihp-sg13g2/spi/config.mk

```diff
@@ -0,0 +1,13 @@
+export DESIGN_NAME = spi
+export PLATFORM    = ihp-sg13g2
+
+export VERILOG_FILES = ./designs/$(PLATFORM)/$(DESIGN_NICKNAME)/src/spi.v
+export SDC_FILE      = ./designs/$(PLATFORM)/$(DESIGN_NICKNAME)/spi.sdc
+
+# Adders degrade GCD
+export ADDER_MAP_FILE :=
```

Comment:
Not gcd

Reply:
unresolved - it sounds like both lines can be removed.

Reply:
done